feat: :sparkles: decision on why we use or recommend CC0 for data (#194)
seedcase-project/decisions: why-cc0-data/index.qmd

---
title: "Why CC0 for data"
description: "Given that we work with and manage data, having a license for the data can be helpful. This post discusses why we recommend (and use) the CC0 license for data."
date: "2025-08-01"
categories:
- license
---

::: content-hidden
Use other decision posts as inspiration to writing these. Leave the
content-hidden sections in the text for future reference.
:::

## Context and problem statement

::: content-hidden
State the context and some background on the issue, then write a
statement in the form of a question for the problem.
:::

The license of the data can help alleviate and solve many issues that
arise when working with data. And while our projects involve processing
and managing data, we aren't always the owners of the data so we can't
or aren't able to make decisions on the license to use. However, we
still would like to have a license that we recommend to others or that
we use ourselves when we are the owners of the data. There are many
licenses available, but given we are a project that follows open
principles, we want to use a license that is open. So the question is:

> **What open license should we use for our own data and that we recommend

Reviewer: ```suggestion
**What open license should we use for our own data and recommend
```

to others?**

## Decision drivers

::: content-hidden
List some reasons for why we need to make this decision and what things
have arisen that impact work.
:::

It needs or should be:

-   A valid [open license](https://opendefinition.org/licenses/)
-   Easy to understand.

> -   Be fairly widely used or recommended.

Reviewer: ```suggestion
-   Fairly widely used or recommended.
```

-   Applicable to a wide range of data.
-   A globally applicable license.
-   Applicable to data itself, not the database architecture.
-   Generally applicable or specific to scientific/research data.

## Considered options

::: content-hidden
List and describe some of the options, as well as some of the benefits
and drawbacks for each option.
:::

> Licensing of data is quite a complex topic, nor are we lawyers. So we

Reviewer: ```suggestion
Licensing of data is quite a complex topic, and we are not lawyers. So we
```

need to select licenses that are well established, widely used, and
simple. For instance, facts (someone's weight in kilograms) cannot be
copyrighted and since generally research/scientific data is fact-based,
it might not even be copyrightable, depending on the jurisdiction. For
*data* (not database structure or organisation), there are two main
"providers" of open licenses: Open Data Commons and Creative Commons.
Within these, there are share-alike, non-commercial, and non-derivative
versions of their licenses. There are not generally recommended for
scientific data, as they can substantially limit the practical use of
the data, even for purely academic purposes and even if everything is
legal and applicable. For those reasons, we won't consider them.

> From post, we consider these options:

Reviewer: I'm not sure I've ever hear the expression 'From post' used in this way. Is it a Canadian expression?
